plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -560,7 +561,7 @@ private ConnectorTableMetadata doGetTableMetadata(ConnectorSession session, Sche
                 .orElseThrow(() -> new TableNotFoundException(tableName));
 
         if (isIcebergTable(table) || isDeltaLakeTable(table)) {
-            throw new TrinoException(HIVE_UNSUPPORTED_FORMAT, format("Not a Hive table '%s'", tableName));
```

Comment:
should we remove HIVE_UNSUPPORTED_FORMAT  now?

Reply:
The error code still has about `10` usages and is used to denote compression codec problems , storage format problems.